pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -363,6 +364,17 @@ func Run() {
 	zedagentCtx.subNetworkInstanceMetrics = subNetworkInstanceMetrics
 	subNetworkInstanceMetrics.Activate()
 
+	subAppFlowMonitor, err := pubsub.Subscribe("zedrouter",
+		types.IPFlow{}, false, &zedagentCtx)
+	if err != nil {
+		// XXX
+		log.Errorf("FlowMonitor: create subFlowStatus failed, err %v", err)
```

Comment:
Surrounding code does log.Fatal and crash zedagent when pubsub subscription fails. I think we should do the same here.

Reply:
sure. will change to log.Fatal()